### PR TITLE
Food UI Code

### PR DESCRIPTION
**Pull request by Ashraf7199**
This is my implementation for the food UI.

### CODE REVIEW

File: Food.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <meta name="keywords" content="">
+    <meta name="description" content="">
+    <meta name="page_type" content="np-template-header-footer-from-plugin">
+    <title>Food</title>
+    <link rel="stylesheet" href="main.css" media="screen">
+<link rel="stylesheet" href="Food.css" media="screen">
+    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
+    <script class="u-script" type="text/javascript" src="main.js" defer=""></script>
+    
+    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
```
Comment (CalebObsCode) [inline Food.html:15]: What is the essence of this? why is this being used?

File: Food.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        </a>
+        <h1 class="u-text u-text-custom-color-1 u-text-1">Welcome to Wayne Society</h1>
+        <nav data-position="" class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
+          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px; font-weight: 700;">
+            <a class="u-button-style u-custom-active-border-color u-custom-border u-custom-border-color u-custom-borders u-custom-hover-border-color u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-active-color u-custom-text-color u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link" href="#" style="padding: 2px 0px; font-size: calc(1em + 4px);">
+              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
+              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
```
Comment (CalebObsCode) [inline Food.html:39]: Why are we using svg?

File: Food.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          </div>
+        </nav>
+      </div></header> 
+    
+    
+    <footer class="u-align-center u-clearfix u-footer u-grey-80 u-footer" id="sec-aba3"><div class="u-clearfix u-sheet u-valign-middle u-sheet-1">
+        <p class="u-small-text u-text u-text-variant u-text-1">event</p>
+      </div></footer>
+  </body>
+</html>
```
Comment (herskovitz2000) [inline Food.html:76]: Overall code looks good and has consistent style, but could use some comments to make code easier to read

File: Food.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    
+    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
+    
+    <script type="application/ld+json">{
+		"@context": "http://schema.org",
+		"@type": "Organization",
+		"name": "Wayne",
+		"url": "index.html",
+		"logo": "images/360_F_359190084_yNAOJ5FrCbNHklodaobTM2jsdHEb3mqs.jpg"
+}</script>
```
Comment (erahman99) [inline Food.html:23]: would it not be better to put the JavaScript in a separate file

File: Food.html (hunk 3 above)
Comment (erahman99) [reply Food.html:76]: needs more comments

File: Food.html (hunk 4 above)
Comment (CalebObsCode) [reply Food.html:23]: I agree with erahman, next time let's use external javascript to make the code more readable

**Pull request merged**